docs/source/querying_data/reranking.md

```diff
@@ -0,0 +1,119 @@
+# Reranking
+
+Second-stage reranking is a technique to use AI to gain significant search relevancy (we've seen +5-10% recall in the top 5) for your queries. OpenSearch 2.12 introduces the [Rerank processor](https://opensearch.org/docs/latest/search-plugins/search-pipelines/rerank-processor/), contributed by Aryn, to perform this operation. For more information, visit the [OpenSearch reranking documentation](https://opensearch.org/docs/latest/search-plugins/search-relevance/reranking-search-results/).
+
+## Usage
+
+In order to use the rerank processor, you first need to register and deploy a [Text Similarity Model](https://opensearch.org/docs/latest/ml-commons-plugin/custom-local-models/#cross-encoder-models). If you're using the quickstart containers, one such model comes deployed already. You can get its id (let's call this `reranker_id`) with
```

Comment:
They aren't called Quickstart containers. It's just Sycamore, so instead say "Sycamore includes NAME OF MODEL, and you will need to get it's unique ID to use for reranking. To get its ID...